src/Paystack.php

```diff
@@ -25,6 +27,25 @@ public function useGuzzle()
         $this->use_guzzle = true;
     }
 
+    public function useRoutes(array $routes)
+    {
+        foreach ($routes as $route => $class) {
+            if (! is_string($route)) {
```

Comment:
can we also confirm that it does not shadow an existing route?

Reply:
Done cc @ibrahimlawal